README.md

```diff
@@ -90,6 +90,15 @@ To download the latest stable builds and for repository information
 and installation instructions visit
 [https://osquery.io/downloads](https://osquery.io/downloads/).
 
+We use a simple numbered versioning scheme `X.Y.Z`, where X is a major version, Y is a minor, and Z is a patch.
+We plan minor releases roughly every two months. These relases are tracked on our [Milestones](https://github.com/osquery/osquery/milestones) page. A patch release is used when there are unforseen bugs with out minor release and we need to quickly patch.
```

Comment:
nit/typos: ... when there are **`unforeseen`**  bugs with **`our`** minor release and we need to quickly patch...

Reply:
other nit/typo in the same paragraph: These **`releases`** are tracked on our

Reply:
Thanks!